src/components/treeCard/index.tsx

```diff
@@ -55,6 +56,11 @@ const TreeCard: React.FC<TreeCardProps> = ({ site }) => {
           >
             More Info
           </GreenLinkButton>
+          <ShareButton
+            size="middle"
+            defaultText="Check out this tree I adopted!"
+            link={`map.treeboston.org/tree/${site.id ? site.id : ''}`}
```

Comment:
:memo: Just a random JS trivia that `site.id ? site.id : ''` can be simplified to `site.id || ''` (or `site.id ?? ''` depending on the situation) but this is fine too